arm-visualstudio/2014-04-01-preview/swagger/Csm.json

```diff
@@ -0,0 +1,818 @@
+        "operationId": "Accounts_ListByResourceGroup",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The operation succeeded. The response contains the details of the Visual Studio Team Services account resources linked to the Azure subscription.",
+            "schema": {
+              "$ref": "#/definitions/AccountResourceListResult"
+            }
+         }
+        },
+        "x-ms-examples": {
+            "getResources_List": {
+                "$ref": "../examples/GetResources_List.json"
+            }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/microsoft.visualstudio/account/{accountResourceName}/extension": {
+      "get": {
+        "tags": [
+          "Extensions"
+        ],
+        "summary": "Extensions_ListByAccount",
+        "description": "Gets the details of the extension resources created within the resource group.",
+        "operationId": "Extensions_ListByAccount",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "in": "path",
+            "name": "accountResourceName",
+            "description": "The name of the Visual Studio Team Services account resource.",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The operation succeeded. The response contains all extension resource details for the specified account.",
+            "schema": {
+              "$ref": "#/definitions/ExtensionResourceListResult"
+            }
+         }
+        },
+        "x-ms-examples": {
+            "getExtensionResources_List": {
+                "$ref": "../examples/GetExtensionResources_List.json"
+            }
+        }		
+      }
+    },    
+    "/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/microsoft.visualstudio/account/{accountResourceName}/extension/{extensionResourceName}": {
+      "put": {
+        "tags": [
+          "Extensions"
+        ],
+        "summary": "Extensions_Create",
+        "description": "Registers the extension with a Visual Studio Team Services account.",
+        "operationId": "Extensions_Create",
```

Comment:
PUT is only intended to be used for new extension resources.  PUT/PATCH have different behavior in our API.